### PR TITLE
project19-Ati

### PR DESCRIPTION
**Pull request by takrami**
API: https://api-happy-thought.herokuapp.com/

LIVE: https://jovial-jackson-97a3e9.netlify.app/

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+const Thought = mongoose.model("Thought", thoughtSchema);
+
+app.get("/", (req, res) => {
+  res.send("Hello world");
+});
+
+app.get("/thoughts");
+app.get("/thoughts", async (req, res) => {
+  const thoughts = await Thought.find();
```
Comment (KarolCislowski) [inline backend/server.js:54]: The messages supposed to be sorted from the newest.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please repair the sorting of your thoughts.

File: backend/server.js (hunk 1 above)
Comment (takrami) [reply backend/server.js]: Hey :) Thanks for your comment. It's sorted now.

**Review by KarolCislowski — APPROVED** [review]
Great API 👍🏻

**Pull request closed without merging**